mlflow/system_metrics/metrics/cpu_monitor.py

```diff
@@ -0,0 +1,20 @@
+"""Class for monitoring CPU stats."""
+
+import psutil
+
+from mlflow.system_metrics.metrics.base_metrics_monitor import BaseMetricsMonitor
+
+
+class CPUMonitor(BaseMetricsMonitor):
+    """Class for monitoring CPU stats."""
+
+    def __init__(self, name="cpu"):
+        super().__init__(name)
+
+    def collect_metrics(self):
+        # Get CPU metrics.
+        cpu_percent = psutil.cpu_percent()
+        self._metrics["cpu_percentage"].append(cpu_percent)
```

Comment:
nit, probably want to use self.metrics instead of reaching around to the private var

Reply:
good call, will fix